using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md

```diff
@@ -41,7 +110,9 @@ Audience.signalWithData(traits, visitorProfileCallback);
 {% tab title="iOS" %}
 ### signalWithData
 
-On iOS, UUID is persisted in `NSUserDefaults`.
+This API sends Audience Manager a signal with traits and returns the matching segments for the visitor in a callback. 
+
+Audience manager sends the AAM UUID in response in initial signal call. AAM UUID is persisted in `NSUserDefaults` and sent by SDK in all subsequent signal requests. If available, Experience Cloud ID (MID) is also sent in each signal request along with DPID and DPUUID. The visitor profile that AAM returns is saved in `NSUserDefaults` and updated with every signal call.
```

Comment:
- In the 1st sentence, please capitalize the "m" in "Manager".
- Replace the "in" with a "to" ("...to an initial signal call.)
- In the 2nd sentence, add "is" between "and" and "sent" and add "the" between "by" and "SDK".

In the 3rd sentence:
- Add "the" before "Experience Cloud..."
- Remove "along"
- Add "the" between "with" and DPID and another "the" before "DPUUID.
The sentence should be worded like this:
"If available, the Experience Cloud ID (MID) is also sent in each signal request with the DPID and the DPUUID. "